integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/predicate/ExistsPredicateIT.java

```diff
@@ -4,7 +4,7 @@
  * License: GNU Lesser General Public License (LGPL), version 2.1 or later
  * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
  */
-package org.hibernate.search.integrationtest.backend.elasticsearch.tmp;
+package org.hibernate.search.integrationtest.backend.tck.search.predicate;
```

Comment:
So now we have an `ExistsPredicateIT` and an `ExistsSearchPredicateIT`... That's messy :/